9.4/paper.tex

```diff
@@ -577,7 +577,7 @@ \subsection{Advances in matrix-free infrastructure}\label{sec:mf}
   taking two additional \texttt{std::function} arguments. The first function
   defines the operation to be scheduled on the vector entries before the
   matrix-vector product touches them, and the second what happens
-  afterwards. The new features also include a renumbering to maximize data
+  afterward. The new features also include a renumbering to maximize data
```

Comment:
Interesting :-) https://grammarist.com/usage/afterward-afterwards/

Reply:
I wouldn't have noticed this, but got it back when running the manuscript through [grammarly](grammarly.com). I assumed that American English was the preference.

Reply:
Interesting. Yes, AE it is. Thanks for teaching all of us how to use a new tool!